docs/faq.md

```diff
@@ -192,3 +192,41 @@ In the EarlGrey test target, import the application's app delegate header. In `s
     [delegate resetApplicationForTesting];
 }
 ```
+
+**Why does the screen appear frozen for 30 seconds?**
```

Comment:
ironically 30 seconds is also the timeout for interaction so if there's an ongoing background animation or a synchronization bug that renders the app busy for ~ 30 seconds then it may appear as through screen is frozen but really it's just waiting for idle. Though, in such cases, EarlGrey will fail with a timeout instead.